internal/pkg/github/github.go

```diff
@@ -10,6 +10,7 @@ type Webhook struct {
 	Release    Release    `json:"release"`
 	Repository Repository `json:"repository"`
 	Sender     Sender     `json:"sender"`
+	ChannelID  string     `json:""`
```

Comment:
Do you want to include this property when marshalling to JSON? If not please use: `json:"-"`